src/libprojectM/CMakeLists.txt

```diff
@@ -12,11 +11,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
     add_compile_definitions(
             NOMINMAX
             WIN32_LEAN_AND_MEAN
-            USE_FTGL
-            USE_NATIVE_GLEW
```

Comment:
This flag is required on Windows for GLEW to work properly in some configurations, please keep it to not break Windows builds.